src/Equinox.Core/Tracing.fs

```diff
@@ -0,0 +1,33 @@
+module Equinox.Core.Tracing
+
+open System.Diagnostics
+
+let source = new ActivitySource("Equinox")
+
+[<System.Runtime.CompilerServices.Extension>]
+type ActivityExtensions =
+
+    [<System.Runtime.CompilerServices.Extension>]
+    static member AddTransactAttempt(act: Activity, attempt: int) = act.AddTag("eqx.transact_attempt", attempt)
```

Comment:
attempt might be fine for the tag, but leave the method name as is?

Reply:
🤔 I think let's keep it as "op_attempt" and "transact_attempt" because you might enable spans for `Equinox` but not `Equinox.MessageDb`

Reply:
Hm but a) it's long b) it's unconditional c) the transact_attempt is the outer value within an optional inner op_attempt
I also think in terms of prominence, the transact_attempt should be the first one that you hit, i.e. the notion of the OCC retry loop is pretty fundamental to Equinox
🤔